src/sonic-yang-models/yang-models/sonic-static-route.yang

```diff
@@ -0,0 +1,100 @@
+module sonic-static-route {
+  yang-version 1.1;
+  namespace "http://github.com/Azure/sonic-static-route";
+  prefix sroute;
+
+  import sonic-vrf {
+    prefix vrf;
+  }
+  import ietf-inet-types {
+    prefix inet;
+  }
+  import sonic-types {
+    prefix stypes;
+  }
+
+  organization
+    "SONiC";
+  contact
+    "SONiC";
+  description
+    "STATIC ROUTE yang Module for SONiC OS";
+
+  revision 2022-03-17 {
+    description
+      "First Revision";
+  }
+
+  container sonic-static-route {
+    container STATIC_ROUTE {
+      description
+        "STATIC_ROUTE part of config_db.json";
+      list STATIC_ROUTE_LIST {
+        key "vrf_name prefix";
+        leaf prefix {
+          type stypes:sonic-ip4-prefix;
+          description
+            "prefix is the destination IP address, as key";
+        }
+        leaf vrf_name {
+          type union {
+            type string {
+              pattern 'default';
+            }
+            type leafref {
+              path "/vrf:sonic-vrf/vrf:VRF/vrf:VRF_LIST/vrf:name";
+            }
+          }
+          description
+            "Virtual Routing Instance name as key";
+        }
+        leaf nexthop {
+          type inet:ip-address;

```

Comment:
ECMP not supported? we need comma seperated NH list.

Reply:
Can make the yang file support it. However we would not be able to test it on our solution.

Reply:
frrcfgd already supports ECMP support for static route, please check the ECMP route manually in FRR.